### DISCUSSION TITLE
Is this possible to use SQLCipher configuration via SPM?

### DISCUSSION THREAD

**Discussion by AKoulabukhov** [groue/GRDB.swift#1202] [Q&A]
I see that the only proposed way to use SQLCipher is subspec 'SQLCipher' via CocoaPods. Is there a way to integrate GRDB with **encryption** in a non-cocoa pods project?

**Comment by groue** [comment]
Hello @AKoulabukhov,

No, GRDB does not provide any ready-made solution for integrating GRDB and SQLCipher with SPM.

Actually, SQLCipher is not available on SPM: https://github.com/sqlcipher/sqlcipher/issues/371

Now, some GRDB forks were able to perform such a setup. I can't give precise guidance, but have a look at https://github.com/groue/GRDB.swift/network for inspiration!
